package.json

```diff
@@ -18,7 +18,7 @@
     "pretest": "tsc --strict --skipLibCheck --noEmit test/typecheck.ts",
     "test": "nyc mocha",
     "posttest": "eslint lib test",
-    "test:integration": "mocha test/integration/test.js",
+    "test:integration": "mocha test/integration/test.js --exit",
```

Comment:
Not sure what's hanging around given the process kills, but doesn't work without this.